### DISCUSSION TITLE
Bios settings

### DISCUSSION THREAD

**Discussion by mikeyr0cks** [stevedat/HP-Elitebook-830-G6-Hackintosh#13] [Q&A]
Hi, can you please tell me what bios settings work for you? I have an 830 g6, i7 8565u , ax200, lte fibocom, samsung nvme ssd. It goes into the installer, cant format the ssd. Used another ssd, it installs but when I try selecting “install macos” after the recovery install, it stays frozen. Is it an ssd issue?

**Comment by pit5bul** [comment]
i replaced my NVMe because of the same reason,.. installer would just
fail...

**Reply by mikeyr0cks** [reply]
I searched the whole internet for a fix for the pm981a. Sadly no solution. I will replace the ssd with a supported one and rock on.

**Reply by stevedat** [reply]
yup, even fixed, not good performance as it requires kexts, quirks and new OS updates might break it. You better buy a suitable ssd to replace.
